setup.cfg

```diff
@@ -23,15 +23,6 @@ package_dir=
     =src
 packages = find:
 python_requires = >= 3.7
-install_requires =
```

Comment:
`setup.cfg` still needs a list of dependencies so the installed package pulls them in when a user pip/conda installs. It should only be the high-level dependencies which appear to be astropy, dxchange, and tomopy, but you know better.

Reply:
added the package from runtime environment file